lib/requirements/tag_checker.rb

```diff
@@ -19,18 +18,5 @@ def pre_update_issues(params)
     def pre_publish_issues
       pre_update_issues(edition.tags.symbolize_keys)
     end
```

Comment:
It seems like this should iterate through tags for pre_publish and pre_preview issues

Reply:
I've now added methods for pre_publish and pre_update.

pre_preview_issues isn't currently a thing when it comes to tags, so it felt a little odd adding it. I can see it coming in useful later though, so have added some methods for it that return no issues at the moment.